apps/docs/content/guides/auth/migrations/auth0.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Migrate from Auth0 to Supabase Auth'
+subtitle: 'Learn how to migrate your users from Auth0'
+breadcrumb: 'Auth0 migration'
+---
+
+## Migration Strategies
+
+When it comes to migrating your users to a different authentication service, these are some of the considerations that we've seen across other projects.
+
+- How does the cost scale as your user base grows?
+- Does the new auth provider selected cover the same set of features needed (e.g OAuth, password logins, SAML, MFA)?
+- Is it acceptable to have some downtime during the migration?
+- What is your timeline to migrate before terminating the old auth provider?
+
+Depending on your evaluation, you may choose to go with one of the following strategies:
+
+1. Rolling migration
+2. One-off migration
+
+| Strategy | Advantages                                                                   | Disadvantages                                                                                                                                                                                                                                                                                                                                                                                                                |
+| -------- | ---------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Rolling  | 0 downtime, users may need to login again                                    | Need to maintain 2 different auth services, which may be more costly in the short-term.<br/>Need to maintain separate codepaths for the period of the migration.<br/>Some existing users may be inactive and have not signed in with the new provider. This means that you eventually need to backfill those users. However, this is a much smaller-scale one-off migration with lower risks since these users are inactive. |
+| One-off  | No need to maintain 2 different auth services for an extended period of time | Some downtime, users will need to login again. Risky for active users.                                                                                                                                                                                                                                                                                                                                                       |
+
```

Comment:
Perhaps add a quick overview of process

To migrate, follow these three steps

1. Export your user data for password based authentication.
2. Import your password-based users into Supabase auth with a hybrid migration strategy.
3. Update your OAuth configuration to import your passwordless and social-login users

Reply:
hmm i think i've covered it [here](https://github.com/supabase/supabase/pull/27866/files#diff-0800832b547c081d813ae4b275f8c20892e08c046c85ba1132b9016cc9192793R53-R56) already but let me know if it's clearer to move this up